app/client/src/sagas/EvalErrorHandler.ts

```
@@ -287,11 +287,17 @@ export function* evalErrorHandler(
         break;
       }
       case EvalErrorTypes.PARSE_JS_ERROR: {
-        toast.show(`${error.message} at: ${error.context?.entity.name}`, {
+        let errorMessage = error.message;
+
+        if (!!error.context) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)